compiler/AST/AggregateType.cpp

```diff
@@ -1875,8 +1805,6 @@ bool AggregateType::parentDefinesInitializer() const {
 // initializer if --force-initializers has not been thrown (currently), unless
```

Comment:
The comment mentions things that are temporarily false (which hopefully no longer holds?) as well as constructors and --force-initializers.  Might be worth going through this method again to make sure everything is as we expect it to be.